app/models/course.rb

```diff
@@ -148,4 +148,31 @@ def funding
       'fee'
     end
   end
+
+  def publish_sites
+    update_new_to_running
+    update_published_if_running
```

Comment:
Is it really worth having these as two separate updates? I say inline the updates here in as a single update.